tests/integration/test_requirements_pyyaml.py

```
@@ -0,0 +1,241 @@
+class TestPyYAMLDependencyAddition:
+    """Test that PyYAML dependencies were properly added."""
+    
+    @pytest.fixture
+    def requirements_file(self) -> Path:
+        """Return path to requirements-dev.txt."""
+        return Path('requirements-dev.txt')
+    
+    @pytest.fixture
+    def requirements_content(self, requirements_file: Path) -> str:
+        """Load requirements-dev.txt content."""
+        with open(requirements_file, 'r', encoding='utf-8') as f:
```

Ruff (0.14.6)

Unnecessary mode argument

Remove mode argument

(UP015)